scalability_and_performance/ztp_far_edge/ztp-talm-updating-managed-policies.adoc

```diff
@@ -59,6 +59,41 @@ include::modules/cnf-topology-aware-lifecycle-manager-creating-custom-resources.
 [role="_additional-resources"]
 .Additional resources
 
-* For more information about the {cgu-operator} pre-caching workflow, see xref:../../scalability_and_performance/ztp_far_edge/cnf-talm-for-cluster-upgrades.adoc#talo-precache-feature-concept_cnf-topology-aware-lifecycle-manager[Using the container image pre-cache feature].
+* For more information about the {cgu-operator} precaching workflow, see xref:../../scalability_and_performance/ztp_far_edge/cnf-talm-for-cluster-upgrades.adoc#talo-precache-feature-concept_cnf-topology-aware-lifecycle-manager[Using the container image precache feature].
 
 include::modules/cnf-topology-aware-lifecycle-manager-autocreate-cgu-cr-ztp.adoc[leveloffset=+1]
+
+// IBU with LCAO
```

Comment:
I think we should use LCA vs LCAO to align with the product name of the operator